### PR TITLE
Jobs history/general cleanup/refactor

### PR DESCRIPTION
**Pull request by sdunster**
- Try to build complete DOM in one go rather than incrementally
- JS format/lint changes
- some comments

Compared output for following before and after, no visible change:
- https://trainbeacon.ses.nsw.gov.au/Jobs/5745
- https://trainbeacon.ses.nsw.gov.au/Jobs/6009

### CODE REVIEW

File: src/injectscripts/jobs/view.js [hunk 1]
```diff
@@ -103,47 +101,36 @@ document.getElementById("CompleteTeamQuickTextBox").onchange = function() {
...
-      masterViewModel.completeTeamViewModel.primaryActivity(d);
-      masterViewModel.completeTeamViewModel.availablePrimaryTasks.subscribe(function(d) {
-        d.forEach(function(d) {
-          if (d.Name == child) {
-            masterViewModel.completeTeamViewModel.primaryTask(d)
-          }
-        });
-      })
-    }
+  _.each([
```
Comment (sdunster) [inline src/injectscripts/jobs/view.js]: this was a nice win

**Comment by lucanos** [comment]
I'll continue on this after a bit of sleep...

**Comment by lucanos** [comment]
BUG - Showing "Show Older Reports" toggle form "Same Address" section, when that section should show all reports within the 12 month search period.

BUG - "1 Jobjob" in Section Headings.

**Comment by sdunster** [comment]
Fixed Jobjob business but I can't figure out the other bug.

**Pull request merged**